django_guid/integrations/__init__.py

```diff
@@ -0,0 +1,24 @@
+from django.core.exceptions import ImproperlyConfigured
```

Comment:
We discussed this between us, but I'll write it down.

I'd like this code to live in `integrations.py` or `base.py`, and rather imported in `__init__.py`.

Reply:
Moved to base.py